wrapper/lib/output-listener.js

```diff
@@ -20,13 +20,18 @@
  * console.log(listener.contents);
  */
 class OutputListener {
-  constructor () {
+  constructor (streamWriter) {
     this._buff = [];
+    this._streamWriter = streamWriter;
   }
 
   get listener () {
     const listen = function listen (data) {
       this._buff.push(data);
+
+      if (this._streamWriter) {
+        this._streamWriter.write(data);
```

Comment:
I don't believe we need to flush explicitly here, since we aren't corking the steam: https://nodejs.org/api/stream.html#writablecork